art/preprocessing/standardisation_mean_std/standardisation_mean_std_pytorch.py

```diff
@@ -69,14 +73,16 @@ def forward(
     ) -> Tuple["torch.Tensor", Optional["torch.Tensor"]]:
         """
         Apply standardisation with mean and standard deviation to input `x`.
+
+        :param x: Input data for which the gradient is estimated. First dimension is the batch size.
+        :param grad: Gradient value so far.
+        :return: The gradient (estimate) of the defence.
```

Comment:
I think part of this docsting needs updates, the version for tensorflow looks good.